fgpyo/util/tests/test_inspect.py

```diff
@@ -1,16 +1,20 @@
 from typing import Optional
```

Comment:
Should we add a test some of the functions work for classes that are both dataclass and attr?